docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: "3"
 services:
   mqtt-exporter:
     image: kpetrem/mqtt-exporter
+    build: .
```

Comment:
please remove this line :)

Reply:
removed.
Is it because it is a bad idea for you, or because I have to split this up into a separate commit?

Reply:
because the docker-compose is configured to use the image posted in docker hub (see line 4).
However, you can always create a PR to add a docker-compose.dev.yaml changing the "image: ..." line by "build: ."

Reply:
ok. will do.
adding build and removing image will result in image name "<local-git-dir>_mqtt-exporter", keeping the image will use the name from this.
what would your preference?

Reply:
I would prefer to remove the image line. It is easier that way to differentiate the dev version from the one released in docker hub.